### PR TITLE
Add post for improvements to activity log and timeline

### PR DESCRIPTION
**Pull request by adamsilver**

### CODE REVIEW

File: app/posts/manage-teacher-training-applications/2021-02-11-improving-the-activity-log-and-timeline.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+We decided also to look at how notes are displayed in the activity log and timeline. Currently notes appear only as a link, meaning that users always need to click through to read the note.
+
+## What we did
+
+We added the details of interviews and notes so that users can understand what’s happened without leaving the activity log or timeline.
+
+We also moved away from the blue ‘train track’ styling which didn’t work very well, especially on the activity log where it implied a connection between events in different applications.
+
+It also sets us up nicely to use the same application list layout when we introduce search and filter.
```
Comment (johndoates) [inline app/posts/manage-teacher-training-applications/2021-02-11-improving-the-activity-log-and-timeline.md]: I wasn't sure what you meant here. Could you explain a little further?

File: app/posts/manage-teacher-training-applications/2021-02-11-improving-the-activity-log-and-timeline.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## How it works
+
+Everything that happens to an application is shown in the timeline and activity log.
+
+Events are grouped by day in the activity log.
+
+If an event relates to an interview that exists then a link is provided. If the interview has since been cancelled, all events that relate to that interview will not have a link.
+
+A link is still given for notes.
```
Comment (johndoates) [inline app/posts/manage-teacher-training-applications/2021-02-11-improving-the-activity-log-and-timeline.md]: What's the reason for still linking to notes?

**Review by johndoates — CHANGES_REQUESTED** [review]
I've committed a few changes and added a couple of comments.

File: app/posts/manage-teacher-training-applications/2021-02-11-improving-the-activity-log-and-timeline.md (hunk 1 above)
Comment (adamsilver) [reply app/posts/manage-teacher-training-applications/2021-02-11-improving-the-activity-log-and-timeline.md]: The left hand side filter that may likely end up there, works better with the new (non train track) styles visually and is consistent with the application list styles of simple horizontal rules split up by h2s.

File: app/posts/manage-teacher-training-applications/2021-02-11-improving-the-activity-log-and-timeline.md (hunk 2 above)
Comment (adamsilver) [reply app/posts/manage-teacher-training-applications/2021-02-11-improving-the-activity-log-and-timeline.md]: Oh this doesn't make sense to spell out - but the reason is that they do not get deleted at the moment.

Comment (adamsilver) [reply app/posts/manage-teacher-training-applications/2021-02-11-improving-the-activity-log-and-timeline.md]: In fact, this has already been removed I think.

**Pull request merged**